ui/app/adapters/azure/config.js

```diff
@@ -23,4 +23,24 @@ export default class AzureConfig extends ApplicationAdapter {
       };
     });
   }
```

Comment:
Claire, you had mentioned previously only calling updateRecord to better match what is happening on the backend (they "create" a record the moment an azure secret engine is mounted). I could not for the life of me get that flow to work (likely a misunderstanding on my end)—let's chat via zoom if it's a pattern you want me to pursue further.

Reply:
Totally up to you! It was intended to help simplify things, not make them more complicated 😂 

All you'd do is remove the code where you call `this.store.createRecord` in `secrets/backend/configuration/edit.ts` and instead just update + save the model you already fetched a few lines above.